pandas_gbq/gbq.py

```diff
@@ -815,6 +844,27 @@ def read_gbq(
         :class:`google.oauth2.service_account.Credentials` directly.
 
         .. versionadded:: 0.8.0
+    use_bqstorage_api : bool, default False
+        Use the `BigQuery Storage API
+        <https://cloud.google.com/bigquery/docs/reference/storage/>`__ to
+        download query results quickly, but at an increased cost. To use this
+        API, first `enable it in the Cloud Console
+        <https://console.cloud.google.com/apis/library/bigquerystorage.googleapis.com>`__.
+        You must also have the `bigquery.readsessions.create
+        <https://cloud.google.com/bigquery/docs/access-control#roles>`__
+        permission on the project you are billing queries to.
+
+        **Note:** Due to a `known issue in the ``google-cloud-bigquery``
+        package
+        <https://github.com/googleapis/google-cloud-python/pull/7633>`__
+        (fixed in version 1.11.0), you must write your query results to a
+        destination table. To do this with ``read_gbq``, supply a
+        ``configuration`` dictionary.
```

Comment:
We could only allow importing this for versions >1.11.0

Reply:
A few thoughts on this:

* I'd be more comfortable bumping the required version once there's been a few more versions of `google-cloud-bigquery` released past 1.11. I like people to have a few options in case there's a problematic regression in 1.11.1.
* The failure on small results will be quite obvious (`InternalError` stacktrace) if they do try to use with an older version. If someone really wants to try to use it with an older version, they'll find out pretty quickly why they should upgrade, or they really know what they are doing and that they are always going to get results that are large enough for the BigQuery Storage API to read from.
* Someday we might want to make `use_bqstorage_api=True` the default (maybe after the BQ Storage API goes GA?). I think it'd definitely be worth bumping the minimum version at that time.

Reply:
Yes that sounds v reasonable. Agree re making the default True when it's GA (or at least soliciting opinions)

tbc I was suggesting _only_ for using bqstorage do we make the min version 1.11 (probably implemented above when attempting the import). Other versions would still run everything else